docs/models/locations.md

```diff
@@ -0,0 +1,30 @@
+# Locations
+
+Model to save real wold places. 
+
+## BaseLocation
+
+We need at least one of geopoint, radius or shapefile
+
+Inherits:
+```
+mptt.models.MPTTModel
```

Comment:
We can mention here that we are using django-mptt https://github.com/django-mptt/django-mptt for adding `Modified Preorder Tree Traversal` support